# HG changeset patch
# User jwe
# Date 747288879 0
#      Mon Sep 06 04:14:39 1993 +0000
# Node ID ab04b34037f0c177361909845a3c60074baec451
# Parent  998cbb0a9e5188e5ef5702421a7b94e6df026824
[project @ 1993-09-06 04:14:39 by jwe]
(DAE::integrate (Vector&, Matrix&, Vector&)): Add missing parameter in
call to integrate() if no critical points are specified.

diff --git a/liboctave/DASSL.cc b/liboctave/DASSL.cc
--- a/liboctave/DASSL.cc
+++ b/liboctave/DASSL.cc
@@ -445,13 +445,13 @@ DAE::integrate (const Vector& tout, Matr
 		    }
 		}
 
 	      if (do_restart)
 		force_restart ();
 	    }
 	}
       else
-	retval = integrate (tout);
+	retval = integrate (tout, xdot_out);
     }
 
   return retval;
 }
